master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -43,7 +43,7 @@ spec:
 | logSeveritySys | The log severity above which logs are sent to the syslog. Set to `NONE` for no logging to syslog. | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive) | string | `INFO` |
 | logSeverityFile| The log severity above which logs are sent to the log file. | Same as `logSeveritySys` | string | `INFO` |
 | logSeverityScreen | The log severity above which logs are sent to the stdout. | Same as LogSeveritySys | string | `INFO` |
-| ignoreLooseRPF\* | Set to `true` to allow Felix to run on systems with loose reverse path forwarding (RPF). | boolean | boolean | `false` |
+| ignoreLooseRPF | Set to `true` to allow Felix to run on systems with loose reverse path forwarding (RPF). **Warning**: setting `ignoreLooseRPF` to `true` will allow containers on the same host to see each other's IP addresses. | boolean | boolean | `false` |
```

Comment:
The warning's not quite right, it's not that they can see each other's IPs, it's that they may be able to spoof (use) each other's IPs (if they have the permissions needed to send packets with arbitrary source IPs).

In many cases, containers will be too locked down to allow them to spoof, but OpenStack VMs _can_ spoof their IPs, as can privileged docker containers and docker containers with specific capabilities granted.

Reply:
OK good to know @fasaxc  . I've edited the warning, let me know if this describes the situation any better.